ACLPManager/manager
# upcoming: [DI-20595] - ACLP Supported regions per service type

## Description 📝
ACLP supports a limited set of regions per service type and capability based on LaunchDarkly config. Regions are now correctly filtered according to the capability they belong to and the supported regions fetched from launchDarkly.

## Changes  🔄
- Added new key for supported regions in aclpServiceType Flag.
- Validated the region ids from flag with all the regions fetched from api.
- Filtered the service type specific regions using corresponding capabilities.
- Added relevant unit test to verify.


## Target release date 🗓️
<>

## Preview 📷
| Before  | After   |
| ------- | ------- |
| 📷 | 📷 |

## How to test 🧪

### Verification steps
- Navigate to monitor tab.
- Select a dashboard.
- Open networks tab, find the launchDarkly config, see supported region ids in aclpServiceTypeMap, verify that these are same as the region ids listed in 'regions' autocomplete popper list. _Note:_ ids are listed at the end of region option label in autcomplete list.

<details>
<summary> Author Checklists </summary>

## As an Author, to speed up the review process, I considered 🤔

*Check all that apply*


<br/>


## As an Author, before moving this PR from Draft to Open, I confirmed ✅


</details>
